src/transaction/log_applier.c

```diff
@@ -7043,142 +7043,152 @@ la_print_log_arv_header (const char *database_name, LOG_ARV_HEADER * hdr, bool v
+  printf ("%-30s : %ld\n", "Delete count", ha_apply_info.delete_counter);
+  printf ("%-30s : %ld\n", "Schema count", ha_apply_info.schema_counter);
+  printf ("%-30s : %ld\n", "Commit count", ha_apply_info.commit_counter);
+  printf ("%-30s : %ld\n", "Fail count", ha_apply_info.fail_counter);
 
-	  db_datetime_to_string ((char *) timebuf, 1024, &ha_apply_info.last_access_time);
-	  printf ("%-30s : %s\n", "Last access time", timebuf);
-	}
+  if (verbose)
+    {
+      db_datetime_to_string ((char *) timebuf, 1024, &ha_apply_info.start_time);
+      printf ("%-30s : %s\n", "Start time", timebuf);
+    }
 
-      printf ("%-30s : %ld\n", "Insert count", ha_apply_info.insert_counter);
-      printf ("%-30s : %ld\n", "Update count", ha_apply_info.update_counter);
-      printf ("%-30s : %ld\n", "Delete count", ha_apply_info.delete_counter);
-      printf ("%-30s : %ld\n", "Schema count", ha_apply_info.schema_counter);
-      printf ("%-30s : %ld\n", "Commit count", ha_apply_info.commit_counter);
-      printf ("%-30s : %ld\n", "Fail count", ha_apply_info.fail_counter);
+  if (check_replica_info)
+    {
+      replica_time_bound_str = prm_get_string_value (PRM_ID_HA_REPLICA_TIME_BOUND);
+      db_datetime_to_string2 ((char *) timebuf, 1024, &ha_apply_info.log_record_time);
 
-      if (verbose)
+      printf ("\n *** Replica-specific Info. *** \n");
+      if (replica_time_bound_str == NULL)
 	{
-	  db_datetime_to_string ((char *) timebuf, 1024, &ha_apply_info.start_time);
-	  printf ("%-30s : %s\n", "Start time", timebuf);
+	  printf ("%-30s : %d second(s)\n", "Deliberate lag", prm_get_integer_value (PRM_ID_HA_REPLICA_DELAY_IN_SECS));
 	}
 
-      if (check_replica_info)
+      printf ("%-30s : %s\n", "Last applied log record time", timebuf);
+      if (replica_time_bound_str != NULL)
 	{
-	  replica_time_bound_str = prm_get_string_value (PRM_ID_HA_REPLICA_TIME_BOUND);
-	  db_datetime_to_string2 ((char *) timebuf, 1024, &ha_apply_info.log_record_time);
-
-	  printf ("\n *** Replica-specific Info. *** \n");
-	  if (replica_time_bound_str == NULL)
-	    {
-	      printf ("%-30s : %d second(s)\n", "Deliberate lag",
-		      prm_get_integer_value (PRM_ID_HA_REPLICA_DELAY_IN_SECS));
-	    }
-
-	  printf ("%-30s : %s\n", "Last applied log record time", timebuf);
-	  if (replica_time_bound_str != NULL)
-	    {
-	      printf ("%-30s : %s\n", "Will apply log records up to", replica_time_bound_str);
-	    }
+	  printf ("%-30s : %s\n", "Will apply log records up to", replica_time_bound_str);
 	}
     }
 check_applied_info_end:
   if (error != NO_ERROR)
     {
-      printf ("%s\n", db_error_string (3));
+      printf ("ERROR : %s\n\n", db_error_string (3));
     }
-  error = NO_ERROR;
+  return error;
+}
 
-  if (check_copied_info)
+/*
+ * applyinfo_copied_log_info() - 
+ *   return: 
+ */
+int
+la_applyinfo_copied_log_info (const char *database_name, const char *log_path, INT64 page_num, bool verbose,
```

Comment:
I think it is not meaningful function name

Reply:
> function name is ambiguous. I think user can not guess what this function does by its name.
> how about la_print_applied_log_info or la_get_applied_log_info? or try something meaningful.

I understood your reply is related with this reply, right?